inlong-sort/sort-connectors/elasticsearch-6/src/main/java/org/apache/inlong/sort/elasticsearch6/Elasticsearch6BulkProcessorIndexer.java

```diff
@@ -53,32 +49,10 @@ class Elasticsearch6BulkProcessorIndexer implements RequestIndexer {
     }
 
     @Override
-    public void add(DeleteRequest... deleteRequests) {
-        for (DeleteRequest deleteRequest : deleteRequests) {
-            if (flushOnCheckpoint) {
-                numPendingRequestsRef.getAndIncrement();
-            }
-            this.bulkProcessor.add(deleteRequest);
-        }
-    }
-
-    @Override
-    public void add(IndexRequest... indexRequests) {
-        for (IndexRequest indexRequest : indexRequests) {
-            if (flushOnCheckpoint) {
-                numPendingRequestsRef.getAndIncrement();
-            }
-            this.bulkProcessor.add(indexRequest);
-        }
-    }
-
-    @Override
-    public void add(UpdateRequest... updateRequests) {
-        for (UpdateRequest updateRequest : updateRequests) {
-            if (flushOnCheckpoint) {
-                numPendingRequestsRef.getAndIncrement();
-            }
-            this.bulkProcessor.add(updateRequest);
+    public void add(DocWriteRequest<?> request) {
+        if (flushOnCheckpoint) {
+            numPendingRequestsRef.getAndIncrement();
```

Comment:
Will remove `for` cause other problem?

Reply:
> 

It is only a optimization for api call will not cause any problem.